lib/python/ensembl/brc4/runnable/process_gff3.py

```diff
@@ -37,6 +37,9 @@ def param_defaults(self):
                 "pseudogene",
                 "ncRNA_gene",
             ),
+            "non_gene_types": (
```

Comment:
I'm not 100% happy with this key, but I cannot think of an alternative that sounds better to me.